<ui version="4.0" >
 <class>ConfigDialog</class>
 <widget class="QMainWindow" name="ConfigDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="contextMenuPolicy" >
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle" >
   <string>Settings</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="../res/vidalia.qrc" >:/images/32x32/preferences-system.png</iconset>
  </property>
  <property name="iconSize" >
   <size>
    <width>48</width>
    <height>48</height>
   </size>
  </property>
  <property name="toolButtonStyle" >
   <enum>Qt::ToolButtonTextUnderIcon</enum>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <layout class="QVBoxLayout" >
    <property name="margin" >
     <number>9</number>
    </property>
    <property name="spacing" >
     <number>6</number>
    </property>
    <item>
     <widget class="ConfigPageStack" native="1" name="stackPages" >
      <property name="palette" >
       <palette>
        <active>
         <colorrole role="WindowText" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>0</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Button" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>208</red>
            <green>208</green>
            <blue>208</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Light" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Midlight" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>247</red>
            <green>247</green>
            <blue>247</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Dark" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>120</red>
            <green>120</green>
            <blue>120</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Mid" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>160</red>
            <green>160</green>
            <blue>160</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Text" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>0</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="BrightText" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="ButtonText" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>0</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Base" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Window" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>240</red>
            <green>240</green>
            <blue>240</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Shadow" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>0</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Highlight" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>10</red>
            <green>36</green>
            <blue>106</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="HighlightedText" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Link" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>0</red>
            <green>0</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="LinkVisited" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>255</red>
            <green>0</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="AlternateBase" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>247</red>
            <green>247</green>
            <blue>247</blue>
           </color>
          </brush>
         </colorrole>
        </active>
        <inactive>
         <colorrole role="WindowText" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>0</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Button" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>208</red>
            <green>208</green>
            <blue>208</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Light" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Midlight" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>247</red>
            <green>247</green>
            <blue>247</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Dark" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>120</red>
            <green>120</green>
            <blue>120</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Mid" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>160</red>
            <green>160</green>
            <blue>160</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Text" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>0</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="BrightText" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="ButtonText" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>0</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Base" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Window" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>240</red>
            <green>240</green>
            <blue>240</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Shadow" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>0</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Highlight" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>212</red>
            <green>208</green>
            <blue>200</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="HighlightedText" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>0</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Link" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>0</red>
            <green>0</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="LinkVisited" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>255</red>
            <green>0</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="AlternateBase" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>247</red>
            <green>247</green>
            <blue>247</blue>
           </color>
          </brush>
         </colorrole>
        </inactive>
        <disabled>
         <colorrole role="WindowText" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>120</red>
            <green>120</green>
            <blue>120</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Button" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>208</red>
            <green>208</green>
            <blue>208</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Light" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Midlight" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>247</red>
            <green>247</green>
            <blue>247</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Dark" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>120</red>
            <green>120</green>
            <blue>120</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Mid" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>160</red>
            <green>160</green>
            <blue>160</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Text" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>120</red>
            <green>120</green>
            <blue>120</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="BrightText" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="ButtonText" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>120</red>
            <green>120</green>
            <blue>120</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Base" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>240</red>
            <green>240</green>
            <blue>240</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Window" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>240</red>
            <green>240</green>
            <blue>240</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Shadow" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>0</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Highlight" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>10</red>
            <green>36</green>
            <blue>106</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="HighlightedText" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Link" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>0</red>
            <green>0</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="LinkVisited" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>255</red>
            <green>0</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="AlternateBase" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>247</red>
            <green>247</green>
            <blue>247</blue>
           </color>
          </brush>
         </colorrole>
        </disabled>
       </palette>
      </property>
      <property name="contextMenuPolicy" >
       <enum>Qt::NoContextMenu</enum>
      </property>
      <widget class="QWidget" native="1" name="widget" >
       <property name="geometry" >
        <rect>
         <x>0</x>
         <y>0</y>
         <width>100</width>
         <height>30</height>
        </rect>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QDialogButtonBox" name="buttonBox" >
      <property name="orientation" >
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="standardButtons" >
       <set>QDialogButtonBox::Cancel|QDialogButtonBox::NoButton|QDialogButtonBox::Ok</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar" />
  <widget class="QToolBar" name="toolBar" >
   <property name="contextMenuPolicy" >
    <enum>Qt::NoContextMenu</enum>
   </property>
   <property name="movable" >
    <bool>false</bool>
   </property>
   <property name="orientation" >
    <enum>Qt::Horizontal</enum>
   </property>
   <attribute name="toolBarArea" >
    <number>4</number>
   </attribute>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ConfigPageStack</class>
   <extends>QStackedPageWidget</extends>
   <header>config/configpagestack.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../res/vidalia_common.qrc" />
 </resources>
 <connections/>
</ui>
